src/main/java/com/google/sps/servlets/SpoonacularNutritionServlet.java

```diff
@@ -32,15 +32,15 @@
 /** Servlet to take in dish name and return bulk recipe infomation */
 @WebServlet("/dishNutrition")
 public class SpoonacularNutritionServlet extends HttpServlet {
-  static String spoonacularPrefix = "https://api.spoonacular.com/recipes/guessNutrition";
-  static String spoonacularAPIKey = "cd2269d31cb94065ad1e73ce292374a5";
+  private static String spoonacularPrefix = "https://api.spoonacular.com/recipes/guessNutrition";
+  private static String spoonacularAPIKey = "cd2269d31cb94065ad1e73ce292374a5";
```

Comment:
It's better to pass the API key via a commandline flag so that you don't need to expose this in github. Otherwise there is a risk that your API key gets leaked